api/apps/api/src/modules/access-control/projects-acl/project-acl.service.ts

```diff
@@ -92,4 +93,53 @@ export class ProjectAclService implements ProjectAccessControl {
       this.canViewSolutionRoles,
     );
   }
+
+  async checkUserIsOwner(userId: string, projectId: string): Promise<void> {
+    const userIsProjectOwner = await this.roles.findOne({
+      where: {
+        projectId,
+        userId,
+        roleName: Roles.project_owner,
+      },
+    });
+    if (!userIsProjectOwner) {
+      throw new ForbiddenException();
```

Comment:
Service layer doesn't know about REST or HTTP - it cannot assume the consumer is using those. 

consider:
`async isOwner(userId: string, projectId: string): Promise<Permit>`

Reply:
What I understood in this [comment](https://github.com/Vizzuality/marxan-cloud/pull/640#discussion_r754446378) by @ikas was that it was better to do the checks for the ownership of the projects inside each method on the service rather than the controller, thus avoiding repetition in the controller and having the same effect in the end.

Also, if I move this again to the controller by returning the boolean/Permit, wouldn't that mean that I would need to add mixed return types? Like if the method fails and returns a boolean instead of the actual purpose of the method, would it be a little bit confusing?

Reply:
@rubvalave yes, you should do the checks inside the service, but you should not throw a `ForbiddenException` :)

Like @kgajowy mentioned, it doesn't make sense to throw HTTP exceptions (like `ForbiddenException`) from inside the service, since the caller might not be a controller, and it might not be prepared to handle it.

Instead, you can just return `false` here (aka `Denied`, from the `Permit` type), and on the controller, if you receive `false`, then you throw the `ForbiddenException` :+1:

Reply:
I think I understood it better now and I did the appropriate movements regarding this (or so I think) by moving the `ForbiddenException` into the controller after passing the check in the service.

Any comment about this would be much appreciated.

Reply:
looks good, great job :+1: